### ISSUE TITLE
Check feed and enclosures working

### ISSUE THREAD

**Issue by kookster** [PRX/feeder.prx.org#315]
We had an incident where a bad enclosure prefix made all downloads fail.
It wasn't detected for a couple of days.
Wondering if, like the feed monitoring we do for some shows with newrelic, we could build an enclosure and feed monitor to make sure they return rss and mp3s?

**Comment by IcusFMagni** [comment]
Current Point value is for researching solutions.

**Comment by cavis** [comment]
At a minimum, we should add a validation that:

- Looks url-ish
- Doesn't have multiple `http`s in there
- Doesn't have duplicate redirect hosts, maybe?
- Bonus: if there's a published ep, make a request for it and follow redirects to DTR (using an obviously bot-ish UA)

**Comment by kookster** [comment]
if it's a test `head` request, will never get counted

**Comment by cavis** [comment]
Good point!  Also, maybe we don't even need a published ep... if we threw just the root `dovetail.prxu.org` after the prefix?  Maybe?

**Comment by cavis** [comment]
Okay so: make a HEAD request when the prefix changes, to make sure it follows through 302s correctly, and lands on a 200 from `dovetail.prxu.org` or something.

(Do we need an actual MP3 for this???)
